tests/07-remove.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+test_description='Test the remove command'
+
+WHEREAMI=$(dirname "$(realpath "$0")")
+. $WHEREAMI/setup.sh
+
+test_expect_success 'cabin remove tbb mydep toml11' '
+    mkdir remove_test &&
+    cp "$WHEREAMI/../cabin.toml" remove_test/cabin.toml &&
```

Comment:
This depends on the project cabin.toml, which is not ideal as modifications would affect tests. You can do `cabin init & echo <some deps> >> cabin.toml` (let's avoid network access on tests via `cabin add` for now).